packages-exp/auth-exp/src/core/user/reauthenticate.test.ts

```diff
@@ -47,89 +59,91 @@ describe('src/core/user/reauthenticate', () => {
   });
 
   it('should error if the idToken is missing', async () => {
-    await expect(
-      _reauthenticate(
-        user,
-        Promise.resolve(({
-          ...TEST_ID_TOKEN_RESPONSE,
-          idToken: undefined
-        } as unknown) as IdTokenResponse)
-      )
-    ).to.be.rejectedWith(
+    stub(credential, '_getReauthenticationResolver').returns(
+      Promise.resolve(({
+        ...TEST_ID_TOKEN_RESPONSE,
+        idToken: undefined
+      } as unknown) as IdTokenResponse)
```

Comment:
You don't need the extra paren
`as unknown as IdTokenResponse`

Reply:
done